tests/Pay/Adapter/StripeTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Utopia\Tests;
+
+use Utopia\Pay\Adapter\Stripe;
+use PHPUnit\Framework\TestCase;
+
+
+class StripeTest extends TestCase {
+
+    private Stripe $stripe;
+
+    protected function setUp(): void
+    {
+        $secretKey = $_SERVER['STRIPE_SECRET'] ?? 'sk_test_4eC39HqLyjWDarjtT1zdp7dc';
```

Comment:
Let's make this ENV var

Reply:
https://www.php.net/manual/en/function.getenv.php might be more verbose and modern.